conf/messages

```diff
@@ -135,8 +135,12 @@ cases.attachment.upload.error.restrictionSize = Your file will not upload becaus
 cases.attachment.upload.error.mustSelect = You must select a file
 cases.attachment.upload.error.fileType = Your document will not upload because it''s not a valid file type
 
-
-
+case.sample.sample-status-header=Sample status
+case.sample.change-sample-status=Change sample status
```

Comment:
Are both these labels used (lines 139 and 140) or are you just copying an established "pattern"?

Reply:
They are both used, in slightly different places